setup/verify_env.py

```diff
@@ -4,9 +4,11 @@
 from pathlib import Path
 from pip._internal.utils.misc import get_installed_distributions
 
+SETUP_DIRECTORY = Path(__file__).resolve().parent
+
 IS_CHROME = False
 IS_FIREFOX = False
-SMOKE_TEST_SUITE = os.path.join(".", "setup", "verify_setup.robot")
+SMOKE_TEST_SUITE = os.path.join(SETUP_DIRECTORY, "verify_setup.robot")
```

Comment:
With `pathlib` you can do:
```suggestion
SMOKE_TEST_SUITE = SETUP_DIRECTORY / "verify_setup.robot"
```

Reply:
we originally had pathlib in use but then in windows (not in any of our own test machines) we had lot's of issues with pathlib so it was reversed back to os.path instead since that was more stable.